### PR TITLE
build: update bindings

### PR DESCRIPTION
**Pull request by amaanq**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -2,9 +2,6 @@ root = true
 
 [*]
 charset = utf-8
-end_of_line = lf
-insert_final_newline = true
-trim_trailing_whitespace = true
```
Comment (calebdw) [inline .editorconfig:7]: @amaanq, why remove these? Are they not sane defaults (unix style)?

File: package.json [hunk 2]
```diff
@@ -51,31 +51,30 @@
...
-    "eslint": "^9.4.0",
+    "eslint": "^9.9.1",
     "eslint-config-google": "^0.14.0",
     "prebuildify": "^6.0.1",
     "tree-sitter-cli": "^0.23.0"
   },
   "scripts": {
     "install": "node-gyp-build",
-    "prebuildify": "prebuildify --napi --strip",
-    "generate": "for dir in php php_only; do cd $dir && tree-sitter generate --no-bindings && cd ..; done",
```
Comment (calebdw) [inline package.json:75]: Perhaps I'm doing something wrong/inefficient, but I use this `npm run generate` command to quickly generate both parsers. Is there a better one-liner command to generate the parsers when I'm trying to test my changes?

File: .editorconfig (hunk 1 above)
Comment (amaanq) [reply .editorconfig:7]: Forcing one style is incorrect, someone developing a grammar for something Windows-specific might get peeved off

File: package.json (hunk 2 above)
Comment (amaanq) [reply package.json:75]: oh, honestly that might be fine to leave in there. the rationale is that the scripts in here should *solely* pertain to the npm bindings/js files

Comment (amaanq) [reply package.json:75]: (feel free to add that one back, I get that it's annoying having to run that manually in multi-grammar repos)

Comment (calebdw) [reply package.json:75]: Actually, I suppose the makefile could be used for this, `make generate` or just `make` works just fine for me

Comment (amaanq) [reply package.json:75]: yeah that would probably be better actually - feel free to update it or I can in a bit

Comment (calebdw) [reply package.json:75]: Done :smile:

**Pull request merged**